roboteam_ai/src/control/shotControllers/ShotController.cpp

```diff
@@ -39,6 +39,7 @@ ShotData ShotController::getShotData(world::Robot robot, Vector2 shotTarget, boo
        if (hasBall && !genevaIsTurning) {
            shotData = shoot(robot, shotTarget, chip);
        } else if (hasBall && genevaIsTurning) {
+            backDown(robot, currentDesiredGeneva);
```

Comment:
why would you move slower? just go to the ball, contest it and wait for geneva instead.
that would be my idea.
do with it what you want <3 me

Reply:
Ah this is fine, it's being refactored anyways